src/test/java/org/sonarsource/sonarlint/ls/mediumtests/ConnectedModeMediumTests.java

```diff
@@ -849,7 +850,7 @@ void shouldChangeIssueStatus() {
     assertThat(client.shownMessages).contains(new MessageParams(MessageType.Info, "New comment was added"));
   }
 
-  @Test
+  @Disabled("Unstable with the latest core")
```

Comment:
This test reliably passes on my machine but fails on the CI.

GIven that this is now mostly handled by the backend, I'm tempted to remove this test altogether.